plugin/storage/badger/spanstore/reader.go

```diff
@@ -150,6 +150,9 @@ func (r *TraceReader) getTraces(traceIDs []model.TraceID) ([]*model.Trace, error
 		}
 		return nil
 	})
+	if len(traces) == 0 {
+		return nil, spanstore.ErrTraceNotFound
```

Comment:
This is incorrect. ErrTraceNotFound is only supposed to be returned from GetTrace.

Reply:
Sure. I will put it in the public method. Thanks!